pkgs/by-name/tr/transmission_4-macos-bin/package.nix

```
@@ -0,0 +1,54 @@
+{
+  stdenv,
+  lib,
+  fetchurl,
+  _7zz,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "transmission";
+  version = "4.0.6";
+
+  src = fetchurl {
+    url = "https://github.com/transmission/transmission/releases/download/${finalAttrs.version}/Transmission-${finalAttrs.version}.dmg";
+    hash = "sha256-5phX8VLgwvU4TMYDWGw9/ywwyT5nQmM0anAoY+cnfBo=";
+  };
+  sourceRoot = ".";
+
+  nativeBuildInputs = [ _7zz ];
+
+  dontConfigure = true;
+  dontBuild = true;
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/Applications
+    cp -r Transmission/Transmission.app $out/Applications
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "Fast, easy and free BitTorrent client";
+    longDescription = ''
+      Transmission is a BitTorrent client which features a simple interface
+      on top of a cross-platform back-end.
+      Feature spotlight:
+        * Uses fewer resources than other clients
+        * Native Mac, GTK and Qt GUI clients
+        * Daemon ideal for servers, embedded systems, and headless use
+        * All these can be remote controlled by Web and Terminal clients
+        * Bluetack (PeerGuardian) blocklists with automatic updates
+        * Full encryption, DHT, and PEX support
+    '';
+    homepage = "https://www.transmissionbt.com/";
+    downloadPage = "https://github.com/transmission/transmission/releases";
+    changelog = "https://github.com/transmission/transmission/releases/tag/v${finalAttrs.version}";
+    license = with licenses; [
+      gpl2Plus
+      mit
+    ];
+    maintainers = with maintainers; [ fbettag ];
+    platforms = platforms.darwin;
+    sourceProvenance = with lib.sourceTypes; [ binaryNativeCode ];
+  };
```

I think there is value in re-using the `transmission` packages `meta`:
```suggestion
  meta = transmission.meta // (with lib; {
    maintainers = with maintainers; [ fbettag ];
    platforms = platforms.darwin;
    sourceProvenance = with lib.sourceTypes; [ binaryNativeCode ];
  });
```

---

oh wow, this is nice! will be done